web/themes/joinup/templates/custom-form.html.twig

```diff
@@ -0,0 +1,16 @@
+{#
+/**
+ * @file
```

Comment:
> It is in fact only a template for content which is between form tags, so other containers are not needed there, because form tags are still there.
Can you clarify this please?